src/Component/Broadcast/Transport/Internal/ChannelTransport.php

```diff
@@ -76,8 +77,10 @@ public function __construct(Channel $sender, Channel $receiver)
                             unset($this->listeners[$data->channel]);
                         }
                     }
-                } catch (ChannelException) {
+                } catch (ChannelException $e) {
                     // Channel was closed, we can break the loop
+                    // this is still hanging the SSE connection when we close the broadcast server, not sure how to propagate it
```

Comment:
This is what's blocking progress right now. If I start a connection to a broadcast server and then close it, it will successfully reconnect if I try to send a message to a channel. If I start listening a channel and streaming that output to SSE, I don't know how to stop the SSE connection.